src/hooks/useColonyMembersSorting.ts

```diff
@@ -0,0 +1,59 @@
+import { useState, useMemo } from 'react';
+import Decimal from 'decimal.js';
+
+import { Watcher, Contributor } from '~types';
+
+import { SortingMethod } from '~gql';
+
+const useColonyMembersSorting = (
```

Comment:
If this is really just a contributors sorting hook, why not call it `useSortedContributors` or similar?